### PR TITLE
EZP-31672: Rebranding

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31672
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/scss/_login.scss [hunk 1]
```diff
@@ -36,14 +39,20 @@
         &-input {
             border: none;
         }
-
+
```
Comment (dew326) [inline src/bundle/Resources/public/scss/_login.scss:42]:
```suggestion

```

File: src/bundle/Resources/public/scss/_buttons.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
         box-shadow: none;
         background-color: $ez-white;
         border-color: transparent;
-        color: #007DBE;
+        color: #131C26;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_buttons.scss:28]: why not in variable?

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig [hunk 3]
```diff
@@ -1,18 +1,19 @@
 {% extends '@ezdesign/ui/layout.html.twig' %}
 
+
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:3]:
```suggestion
```

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig [hunk 4]
```diff
@@ -1,18 +1,19 @@
...
+        <div class="row error-page">
+            <h2 class="error-page__title">{{ 'error.page.403.title'|trans|desc('You don\'t have permission') }}</h2>
+            <div class="error-page__messages">                                                                                                      
+                <p class="error-page__message error-page__message--title">{{ 'error.page.403.sub_title'|trans|raw|desc('Not possible to retrieve or access this page.') }}</p>
+                <p class="error-page__message error-page__message--actions">{{ 'error.page.403.actions'|trans({ '%dashboard%': path('ezplatform.dashboard'), '%search%': path('ezplatform.search') })|raw|desc('Please contact your Administrator to request access to this content or modify your profile.<br/>Otherwise, go back to <a class="error-page__link" href="%dashboard%">dashboard</a> or <a class="error-page__link" href="%search%">search</a> for another Content item.') }}</p>
+            </div>
         </div>
     </div>
-</div>
+<div>
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:18]:
```suggestion
</div>
```

File: src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig [hunk 5]
```diff
@@ -9,17 +9,16 @@
...
+            <h2 class="error-page__title">{{ 'error.page.404.title'|trans|desc('Page not found') }}</h2>
+            <div class="error-page__messages">
+                <p class="error-page__message error-page__message--title">{{ 'error.page.404.sub_title'|trans|raw|desc('You probably weren\'t expecting to land here.') }}</p>
+                <p class="error-page__message error-page__message--content">{{ 'error.page.404.message'|trans|raw|desc('The page you requested was not found.<br/> It might have been deleted or it is temporarily unavailable.') }}</p>
+                <p class="error-page__message error-page__message--actions">{{ 'error.page.404.actions'|trans({ '%dashboard%': path('ezplatform.dashboard'), '%search%': path('ezplatform.search') })|raw|desc('Check the URL for a typo and try again.<br/>Otherwise, go back to the <a class="error-page__link" href="%dashboard%">dashboard</a> or <a class="error-page__link" href="%search%">search</a> for another Content item.') }}</p>
+            </div>
         </div>
     </div>
-</div>
+<div>
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig:23]:
```suggestion
</div>
```

File: src/bundle/Resources/public/scss/_error-page.scss [hunk 6]
```diff
@@ -1,23 +1,49 @@
...
-    &__messages {
-        max-width: 596px;
-        margin: 3rem auto auto;
-    }
+        &__message {
+            text-align: center;
+            font-size: calculateRem(16px);
+
+            &--title {
+                margin: calculateRem(70px) auto calculateRem(70px);
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_error-page.scss:28]: `margin: calculateRem(70px) auto;`

**Comment by lucasOsti** [comment]
email
login
reset_msg

File: src/bundle/Resources/public/scss/_error-page.scss [hunk 7]
```diff
@@ -1,23 +1,49 @@
...
+                margin-top: calculateRem(40px);
+                font-size: calculateRem(21px);
+                line-height: calculateRem(38px);
+            }
+        }
     }
 
     &__link {
-        text-decoration: underline;
+        color: #0058BA;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_error-page.scss:46]: why not variable?

File: src/bundle/Resources/public/scss/_login.scss (hunk 1 above)
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_login.scss:42]: no spaces here (no prettier?)

File: src/bundle/Resources/public/scss/_login.scss [hunk 8]
```diff
@@ -36,14 +39,20 @@
...
         &-wrapper {
             width: 18rem;
             height: 90%;
             display: flex;
             flex-direction: column;
             justify-content: center;
         }
+
+        &-logo-description {
+            margin: calculateRem(18px) auto calculateRem(40px) auto;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/scss/_login.scss:52]: `margin: calculateRem(18px) auto calculateRem(40px);`

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig (hunk 3 above)
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:3]: unnecessary new line

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig [hunk 9]
```diff
@@ -1,18 +1,19 @@
 {% extends '@ezdesign/ui/layout.html.twig' %}
 
+
+{% block navigation %}{% endblock %}
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:4]:
```{% block navigation %}{% endblock %}

```
add new line after

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig (hunk 4 above)
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:18]: should be closing tag

File: src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig (hunk 5 above)
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig:23]: should be closing tag

File: src/bundle/Resources/translations/messages.en.xliff [hunk 10]
```diff
@@ -62,10 +62,15 @@
         <note>key: authentication.username</note>
       </trans-unit>
       <trans-unit id="b1b04388859d233f757e4f6376af3fc317d4baa6" resname="base.footer_copyright">
-        <source>eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</source>
-        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.</target>
+        <source>eZ Platform ©1999-%current_year% %ez_link_open%Ibexa AS%ez_link_close% and others.</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:65]: I believe:
```suggestion
        <source>eZ Platform ©1999-%current_year% %ez_link_open%Ibexa%ez_link_close% and others.</source>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 11]
```diff
@@ -62,10 +62,15 @@
...
+        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%Ibexa AS%ez_link_close% and others.</target>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:66]:
```suggestion
        <target state="new">eZ Platform ©1999-%current_year% %ez_link_open%Ibexa%ez_link_close% and others.</target>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 12]
```diff
@@ -152,18 +157,23 @@
...
+        <target state="new"><![CDATA[Please contact your Administrator to request access to this content or modify your profile.<br/>Otherwise, go back to <a class="error-page__link" href="%dashboard%">dashboard</a> or <a class="error-page__link" href="%search%">search</a> for another Content item.]]></target>
         <note>key: error.page.403.actions</note>
       </trans-unit>
       <trans-unit id="2532cb9ce6fdf46258c107f3d20888c506115ef6" resname="error.page.403.message">
         <source><![CDATA[Error 403: you don't have permissions to access this page.<br/>Contact your Administrator to request access to this content.]]></source>
         <target state="new"><![CDATA[Error 403: you don't have permissions to access this page.<br/>Contact your Administrator to request access to this content.]]></target>
         <note>key: error.page.403.message</note>
       </trans-unit>
+      <trans-unit id="d0b5337e551b64f59973449bbdcf7914c2102a6c" resname="error.page.403.sub_title">
+        <source>Not possible to retrieve or access this page.</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:170]:
```suggestion
        <source>Cannot retrieve or access this page.</source>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 13]
```diff
@@ -152,18 +157,23 @@
...
+        <target state="new">Not possible to retrieve or access this page.</target>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:171]:
```suggestion
        <target state="new">Cannot retrieve or access this page.</target>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 14]
```diff
@@ -172,13 +182,23 @@
...
+        <target state="new"><![CDATA[The page you requested was not found.<br/> It might have been deleted or it is temporarily unavailable.]]></target>
         <note>key: error.page.404.message</note>
       </trans-unit>
+      <trans-unit id="7596368af317fe78b955e2a3f63c7586900bcd76" resname="error.page.404.messagea">
+        <source>The page you requested was not found. It might have been deleted or it is temporarily unavailable.</source>
+        <target state="new">The page you requested was not found. It might have been deleted or it is temporarily unavailable.</target>
+        <note>key: error.page.404.messagea</note>
+      </trans-unit>
+      <trans-unit id="ffa7f20226d38a989f6c931264bd4cf485b435e6" resname="error.page.404.sub_title">
+        <source>You probably weren't expecting to land here.</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:195]: Do we even need this? It's 404, it's safe to assume nobody wants to land there.

File: src/bundle/Resources/translations/messages.en.xliff [hunk 15]
```diff
@@ -152,18 +157,23 @@
...
       </trans-unit>
+      <trans-unit id="d0b5337e551b64f59973449bbdcf7914c2102a6c" resname="error.page.403.sub_title">
+        <source>Not possible to retrieve or access this page.</source>
+        <target state="new">Not possible to retrieve or access this page.</target>
+        <note>key: error.page.403.sub_title</note>
+      </trans-unit>
       <trans-unit id="b1712cfa4c13874debdcc0f1f4a3b9fa4af3c966" resname="error.page.403.title">
-        <source>You don't have permission for this page.</source>
-        <target state="new">You don't have permission for this page.</target>
+        <source>You don't have permission</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:175]:
```suggestion
        <source>You don't have sufficient permissions to access this page</source>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 16]
```diff
@@ -152,18 +157,23 @@
...
+        <target state="new">You don't have permission</target>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:176]:
```suggestion
        <target state="new">You don't have sufficient permissions to access this page</target>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 17]
```diff
@@ -187,13 +207,18 @@
...
       </trans-unit>
+      <trans-unit id="37404d823f9c9b5ee00f26e4783e5e98d2955750" resname="error.page.default.sub_title">
+        <source>We apologize for the internal server error.</source>
+        <target state="new">We apologize for the internal server error.</target>
+        <note>key: error.page.default.sub_title</note>
+      </trans-unit>
       <trans-unit id="889ba15bcc2f87bfac9b80282f659eaeb7c175d4" resname="error.page.default.title">
-        <source>Something went wrong. Sorry!</source>
-        <target state="new">Something went wrong. Sorry!</target>
+        <source>Something went wrong</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:220]:
```suggestion
        <source>Internal server error</source>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 18]
```diff
@@ -187,13 +207,18 @@
...
+        <target state="new">Something went wrong</target>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:221]:
```suggestion
        <target state="new">Internal server error</target>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 19]
```diff
@@ -187,13 +207,18 @@
...
       </trans-unit>
       <trans-unit id="8338c190462d420926edf71e8d4cddcae710d361" resname="error.page.default.message">
-        <source>Error 500: internal server error. The server was unable to complete your request because of an internal problem.</source>
-        <target state="new">Error 500: internal server error. The server was unable to complete your request because of an internal problem.</target>
+        <source>The server was unable to complete your request because of an internal problem.</source>
+        <target state="new">The server was unable to complete your request because of an internal problem.</target>
         <note>key: error.page.default.message</note>
       </trans-unit>
+      <trans-unit id="37404d823f9c9b5ee00f26e4783e5e98d2955750" resname="error.page.default.sub_title">
+        <source>We apologize for the internal server error.</source>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:215]:
```suggestion
        <source>We apologize for the error.</source>
```

File: src/bundle/Resources/translations/messages.en.xliff [hunk 20]
```diff
@@ -187,13 +207,18 @@
...
+        <target state="new">We apologize for the internal server error.</target>
```
Comment (DominikaK) [inline src/bundle/Resources/translations/messages.en.xliff:216]:
```suggestion
        <target state="new">We apologize for the error.</target>
```

File: src/bundle/Resources/views/themes/admin/account/base.html.twig [hunk 21]
```diff
@@ -13,16 +13,17 @@
...
+                        <div class="ez-login__form-logo-description">{{ 'base.platform_by_ibexa'|trans|desc('platform by Ibexa') }}</div>
                         {%- block content -%} {%- endblock content -%}
                     </div>
                     <div class="ez-login__footer">
                         <span>{{ 'base.footer_copyright'|trans({ 
                                     '%ez_link_open%': '<a class="ez-login__hyperlink" href="http://ez.no">', 
                                     '%ez_link_close%': '</a>', 
                                     '%current_year%': "now"|date("Y") 
-                              }) |desc('eZ Platform ©1999-%current_year% %ez_link_open%eZ Systems AS%ez_link_close% and others.')|raw }}
+                              }) |desc('eZ Platform ©1999-%current_year% %ez_link_open%Ibexa AS%ez_link_close% and others.')|raw }}
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/account/base.html.twig:26]:
```suggestion
                              }) |desc('eZ Platform ©1999-%current_year% %ez_link_open%Ibexa%ez_link_close% and others.')|raw }}
```

File: src/bundle/Resources/public/scss/_buttons.scss (hunk 2 above)
Comment (lucasOsti) [reply src/bundle/Resources/public/scss/_buttons.scss:28]: Variable will be added later after full rebranding now we added by hardcode hex color

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig [hunk 22]
```diff
@@ -1,18 +1,19 @@
...
-            <img src="{{ asset('bundles/ezplatformadminui/img/errors/403.png') }}" />
-        </div>
-        <h2 class="error-page__title">{{ 'error.page.403.title'|trans|desc('You don\'t have permission for this page.') }}</h2>
-        <div class="error-page__messages">
-            <p class="error-page__message">{{ 'error.page.403.message'|trans|raw|desc('Error 403: you don\'t have permissions to access this page.<br/>Contact your Administrator to request access to this content.') }}</p>
-            <p class="error-page__message">{{ 'error.page.403.actions'|trans({ '%dashboard%': path('ezplatform.dashboard'), '%search%': path('ezplatform.search') })|raw|desc('To continue, go back to the <a class="error-page__link" href="%dashboard%">dashboard</a> or <a class="error-page__link" href="%search%">search</a> for another Content item.') }}</p>
+<div class="row error-page-wrapper">
+    <div class="container">
+        <div class="row error-page">
+            <h2 class="error-page__title">{{ 'error.page.403.title'|trans|desc('You don\'t have permission') }}</h2>
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:11]:
```suggestion
            <h2 class="error-page__title">{{ 'error.page.403.title'|trans|desc('You don\'t have sufficient permissions to access this page') }}</h2>
```

File: src/bundle/Resources/public/scss/_error-page.scss (hunk 7 above)
Comment (lucasOsti) [reply src/bundle/Resources/public/scss/_error-page.scss:46]: Variable will be added later after full rebranding now we added by hardcode hex color

File: src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig [hunk 23]
```diff
@@ -1,18 +1,19 @@
...
+            <div class="error-page__messages">                                                                                                      
+                <p class="error-page__message error-page__message--title">{{ 'error.page.403.sub_title'|trans|raw|desc('Not possible to retrieve or access this page.') }}</p>
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig:13]:
```suggestion
                <p class="error-page__message error-page__message--title">{{ 'error.page.403.sub_title'|trans|raw|desc('Cannot retrieve or access this page.') }}</p>
```

File: src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig [hunk 24]
```diff
@@ -9,17 +9,16 @@
...
-        <div class="error-page__messages">
-            <p class="error-page__message">{{ 'error.page.404.message'|trans|raw|desc('Error 404: the page you requested was not found.<br/>It might have been deleted or it is temporarily unavailable.') }}</p>
-            <p class="error-page__message">{{ 'error.page.404.actions'|trans({ '%dashboard%': path('ezplatform.dashboard'), '%search%': path('ezplatform.search') })|raw|desc('Check the URL for a typo and try again.<br/>Otherwise, go back to the <a class="error-page__link" href="%dashboard%">dashboard</a> or <a class="error-page__link" href="%search%">search</a> for another Content item.') }}
-            </p>
+<div class="row error-page-wrapper">
+    <div class="container">
+        <div class="row error-page">
+            <h2 class="error-page__title">{{ 'error.page.404.title'|trans|desc('Page not found') }}</h2>
+            <div class="error-page__messages">
+                <p class="error-page__message error-page__message--title">{{ 'error.page.404.sub_title'|trans|raw|desc('You probably weren\'t expecting to land here.') }}</p>
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig:17]: See above

File: src/bundle/Resources/views/themes/admin/ui/error_page/unknown.html.twig [hunk 25]
```diff
@@ -5,16 +5,15 @@
...
-        </div>
-        <h2 class="error-page__title">{{ 'error.page.default.title'|trans|desc('Something went wrong. Sorry!') }}</h2>
-        <div class="error-page__messages">
-            <p class="error-page__message">{{ 'error.page.default.message'|trans|raw|desc('Error 500: internal server error. The server was unable to complete your request because of an internal problem.') }}</p>
-            <p class="error-page__message">{{ 'error.page.default.actions'|trans|raw|desc('Contact your Administrator and inform them about the time the error occurred, the URL in your browser window, and anything related that you think may have caused the error.<br/>You can also go back or try to reload the application.') }}
-            </p>
+<div class="row error-page-wrapper">
+    <div class="container">
+        <div class="row error-page">
+            <h2 class="error-page__title">{{ 'error.page.default.title'|trans|desc('Something went wrong') }}</h2>
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/error_page/unknown.html.twig:11]:
```suggestion
            <h2 class="error-page__title">{{ 'error.page.default.title'|trans|desc('Internal server error') }}</h2>
```

File: src/bundle/Resources/views/themes/admin/ui/error_page/unknown.html.twig [hunk 26]
```diff
@@ -5,16 +5,15 @@
...
+            <div class="error-page__messages">
+                <p class="error-page__message error-page__message--title">{{ 'error.page.default.sub_title'|trans|raw|desc('We apologize for the internal server error.') }}</p>
```
Comment (DominikaK) [inline src/bundle/Resources/views/themes/admin/ui/error_page/unknown.html.twig:13]:
```suggestion
                <p class="error-page__message error-page__message--title">{{ 'error.page.default.sub_title'|trans|raw|desc('We apologize for the error.') }}</p>
```

File: src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig [hunk 27]
```diff
@@ -5,7 +5,7 @@
 {% block meta %}
     <meta name="LanguageCode" content="{{ language.languageCode }}"/>
 {% endblock %}
-
+a
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/content/edit/edit.html.twig:8]:
```suggestion

```

**Comment by lucasOsti** [comment]
1.) Done
2.) Fixed in https://github.com/ezsystems/ezplatform-ee/pull/145
3.) Problem was browser cache

File: src/bundle/Resources/translations/messages.en.xliff (hunk 10 above)
Comment (inakijv) [reply src/bundle/Resources/translations/messages.en.xliff:65]: @DominikaK footer was double checked by @SylvainGuittard and Legal team. Wireframe shows correct terms based on their review.

File: src/bundle/Resources/translations/messages.en.xliff (hunk 11 above)
Comment (inakijv) [reply src/bundle/Resources/translations/messages.en.xliff:66]: Here I would say it applies the same approach as mentioned above (regarding "Ibexa AS" term) Ping @SylvainGuittard

File: src/bundle/Resources/translations/messages.en.xliff (hunk 10 above)
Comment (DominikaK) [reply src/bundle/Resources/translations/messages.en.xliff:65]: Understood, I just noticed that ibexa.co actually has "Copyright © 2020 Ibexa" in the footer

File: src/bundle/Resources/translations/messages.en.xliff (hunk 14 above)
Comment (inakijv) [reply src/bundle/Resources/translations/messages.en.xliff:195]: Yes, we do need it. It is a standard. We are also aligning error views in Ibexa site and here.

File: src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig (hunk 24 above)
Comment (inakijv) [reply src/bundle/Resources/views/themes/admin/ui/error_page/404.html.twig:17]: We also need this one. It is a standard, as justified in comment above, and for design consistency purposes.

File: src/bundle/Resources/translations/messages.en.xliff (hunk 11 above)
Comment (SylvainGuittard) [reply src/bundle/Resources/translations/messages.en.xliff:66]: Sorry for the late reply on this. Next time do not hesitate to ping me on Slack.
It's fine to go with `Ibexa AS`. 👍

**Pull request merged**